qemu/tests/cgroup.py

```diff
@@ -680,9 +680,7 @@ def cpu_cfs_util():
         timeout = 1.5 * int(params.get("login_timeout", 360))
         # First one
         vms.append(env.get_all_vms()[0])
-        vcpu_thread_pattern = params.get("vcpu_thread_pattern",
-                                         r"thread_id.?[:|=]\s*(\d+)")
-        cpu_pids = vms[0].get_vcpu_pids(vcpu_thread_pattern)
+        cpu_pids = vms[0].get_vcpu_pids(debug=False)
```

Comment:
May I know why set debug=False?

Reply:
@huangyum If we set `debug=True`, this function will print the following output:
```
2019-07-22 05:30:23,844 qemu_monitor     L0359 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Sending command 'query-cpus-fast'
2019-07-22 05:30:23,844 qemu_monitor     L1796 DEBUG| Send command: {'execute': 'query-cpus-fast', 'id': 'MN5XEwcO'}
2019-07-22 05:30:23,847 qemu_monitor     L1752 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Response to 'query-cpus-fast' (re-formated)
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        arch: ppc
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        thread-id: 32958
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        props:
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)            core-id: 0
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        qom-path: /machine/unattached/device[0]/thread[0]
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        cpu-index: 0
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        target: ppc64
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        arch: ppc
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        thread-id: 32959
2019-07-22 05:30:23,848 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        props:
2019-07-22 05:30:23,849 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)            core-id: 1
2019-07-22 05:30:23,849 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        qom-path: /machine/unattached/device[1]/thread[0]
2019-07-22 05:30:23,849 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        cpu-index: 1
2019-07-22 05:30:23,849 qemu_monitor     L1728 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1)        target: ppc64
```
qemu_vm will print it once at the beginning of the test, so I set `debug=False` in the test case side.